inst/tests/tests.Rraw

```diff
@@ -1585,6 +1585,15 @@ test(498.03, setcolorder(DT, 1, after=3), data.table(b=2, c=3, a=1))
 test(498.04, setcolorder(DT, 3, before=1), data.table(a=1, b=2, c=3))
 test(498.05, setcolorder(DT, 1, before=1, after=1), error="Provide either before= or after= but not both")
 test(498.06, setcolorder(DT, 1, before=1:2), error="before=/after= accept a single column name or number, not more than one")
+# skip_absent
+test(498.07, setcolorder(DT, skip_absent='TRUE'), error='TRUE or FALSE')
+test(498.08, setcolorder(DT, skip_absent= 1), error='TRUE or FALSE')
+test(498.09, setcolorder(DT, skip_absent= c(TRUE, FALSE)), error='TRUE or FALSE')
+test(498.10, setcolorder(DT, c('d', 'c', 'b', 'a')), error='non-existing column')
+test(498.11, setcolorder(DT, c('d', 'c', 'b', 'a'), skip_absent=TRUE), data.table(c=3, b=2, a=1))
+test(498.12, setcolorder(DT, 4:1), error='non-existing column')
+test(498.13, setcolorder(DT, 4:1, skip_absent=TRUE), data.table(a=1, b=2, c=3))
+test(498.14, setcolorder(DT, c(1, 1, 2, 3), skip_absent=TRUE), error='!=')
```

Comment:
Please add a test for `setcolorder(DT, c('a', 'b', 'd'), skip_absent=TRUE)`.

The NEWS item currently reads like 'c' will be _dropped_ from the output, which would be bad -- I would expect the above to be equivalent to `setcolorder(DT, c("a", "b"))`.